main.go

```diff
@@ -529,6 +529,65 @@ func main() {
 			return err
 		}
 
+		//
+		// kangaroo-electron
+		//
+		kangarooElectronDescription := "Bundle your holochain app a a standalone electron app with a built-in conductor"
+		kangarooElectronRepositoryArgs := StandardRepositoryArgs("kangaroo-electron", &kangarooElectronDescription)
+		kangarooElectronRepositoryArgs.IsTemplate = pulumi.Bool(true)
+
+		kangarooElectron, err := github.NewRepository(ctx, "kangaroo-electron", &kangarooElectronRepositoryArgs, pulumi.Import(pulumi.ID("kangaroo-electron")))
+		if err != nil {
+			return err
+		}
+		if err = RequireMainAsDefaultBranch(ctx, "kangaroo-electron", kangarooElectron); err != nil {
+		  return err
+		}
+		if err = StandardRepositoryAccess(ctx, "kangaroo-electron", kangarooElectron); err != nil {
+			return err
+		}
+		if err = AdditionalRepositoryAdmin(ctx, "kangaroo-electron", "matthme", kangarooElectron); err != nil {
+			return err
+		}
+		kangarooElectronDefaultRepositoryRulesetArgs := DefaultRepositoryRulesetArgs(kangarooElectron, nil)
+		if _, err = github.NewRepositoryRuleset(ctx, "kangaroo-electron-default", &kangarooElectronDefaultRepositoryRulesetArgs); err != nil {
+			return err
+		}
+		kangarooElectronDefaultRepositoryRulesetArgs.Rules = &github.RepositoryRulesetRulesArgs{
```

Comment:
@ThetaSinner can you give this another look? I'm not proud on this boilerplate here but after struggling for more than an hour trying to find a better pattern and learning that Go neither supports default values for function arguments nor optional arguments I decided to surrender...

Reply:
This looks sensible to me, it's nice and clear